rec_to_nwb/test/e2etests/test_nwbFullGeneration.py

```diff
@@ -30,16 +30,16 @@ def setUpClass(cls):
             process_dio=True,
             process_mda=True,
             process_analog=True,
-            video_directory=str(path) + '/test_data/beans/raw/20190718/'
+            video_directory=str(path) + '/test_data'
```

Comment:
Are you sure that's the correct path?

Reply:
we don`t have specied place to store those files and no clue where it could bo so i assumed  its ok to store it inside test_data for tests